Update convert-disk-storage.md

thiruma0203 opened the pull request:
Minor correction to Powershell scripts, so that it works with the latest Azure Powershell.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

thiruma0203 committed:
Update convert-disk-storage.md

Minor correction to Powershell scripts, so that it works with the latest Azure Powershell.
